data/wy/legislature/Chip-Neiman-b9414bc4-f900-474b-b99c-8f2716419683.yml

```diff
@@ -35,6 +35,11 @@ extras:
   number of grandchildren: '4'
   birth place: Belle Fourche SD
   occupation: Rancher/Small Business
+  leadership:
+  - startYear: '2023'
+    endYear: '2024'
+    position: 'House Majority Floor Leader                       '
+    currentLeader: false
```

Comment:
Not sure what this currentLeader field means but it seems weird that its set to false when this is indeed going to be the House Majority Floor Leader for this session